infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/RustWithCKZG.java

```diff
@@ -102,21 +102,27 @@ public KZGProof computeBlobKzgProof(final Bytes blob, final KZGCommitment kzgCom
     return ckzg4844Delegate.computeBlobKzgProof(blob, kzgCommitment);
   }
 
+  @Override
+  public List<KZGCell> computeCells(final Bytes blob) {
+    // TODO: replace with RustKZG when available
```

Comment:
Where are we tracking this? We are probably gonna forget...

Reply:
it's done, just outdated todo, thank you for catching it